basics/data-import/pinot-stream-ingestion/configure-indexes.md

```diff
@@ -0,0 +1,175 @@
+```json
+{
+  "tableName": "events",
+  "tableType": "REALTIME",
+  "segmentsConfig": {
+    "timeColumnName": "ts",
+    "schemaName": "events",
+    "replicasPerPartition": "1"
+  },
+  "tenants": {},
+  "tableIndexConfig": {
+    "invertedIndexColumns": ["uuid"],
+    "loadMode": "MMAP",
+    "streamConfigs": {
+      "streamType": "kafka",
+      "stream.kafka.consumer.type": "lowlevel",
+      "stream.kafka.topic.name": "events",
+      "stream.kafka.decoder.class.name": "org.apache.pinot.plugin.stream.kafka.KafkaJSONMessageDecoder",
+      "stream.kafka.consumer.factory.class.name": "org.apache.pinot.plugin.stream.kafka20.KafkaConsumerFactory",
+      "stream.kafka.broker.list": "kafka:9092",
+      "realtime.segment.flush.threshold.rows": "0",
+      "realtime.segment.flush.threshold.time": "24h",
+      "realtime.segment.flush.threshold.segment.size": "50M",
+      "stream.kafka.consumer.prop.auto.offset.reset": "smallest"
+    }
+  },
+  "metadata": {
+    "customConfigs": {}
+  }
+}
+```
+
+Navigate to [localhost:9000/#/tenants/table/events_REALTIME](http://localhost:9000/#/tenants/table/events_REALTIME), click on **Edit Table**, paste the next table config, and then click **Save**.
+
+Once you've done that, you'll need to click **Reload All Segments** and then **Yes** to apply the indexing change to all segments.
+
+## Check the index has been applied
+
+We can check that the index has been applied to all our segments by querying Pinot's REST API.
+You can find Swagger documentation at [localhost:9000/help](http://localhost:9000/help).
+
+The following query will return the indexes defined on the `uuid` column:
+
+```bash
+curl -X GET "http://localhost:9000/segments/events/metadata?columns=uuid" \
+  -H "accept: application/json" 2>/dev/null | 
+  jq '.[] | [.segmentName, .indexes]'
+```
+
+##### Output
+
+```json
+[
+  "events__0__1__20220214T1106Z",
+  {
+    "uuid": {
+      "bloom-filter": "NO",
+      "dictionary": "YES",
+      "forward-index": "YES",
+      "inverted-index": "YES",
+      "null-value-vector-reader": "NO",
+      "range-index": "NO",
+      "json-index": "NO"
+    }
+  }
+]
+[
+  "events__0__0__20220214T1053Z",
+  {
+    "uuid": {
+      "bloom-filter": "NO",
+      "dictionary": "YES",
+      "forward-index": "YES",
+      "inverted-index": "YES",
+      "null-value-vector-reader": "NO",
+      "range-index": "NO",
+      "json-index": "NO"
+    }
+  }
+]
+```
+
+We can see from looking at the `inverted-index` property that the index has been applied.
+
+
+We're using the [jq command line JSON processor](https://stedolan.github.io/jq/) to extract the fields that we're interested in.
+
+## Querying
+
+You can now run some queries that filter on the `uuid` column., as shown below:
+
+```sql
+SELECT * 
+FROM events 
+WHERE uuid = 'f4a4f'
+LIMIT 10
+```
+
+You'll need to change the actual `uuid` value to a value that exists in your database, since the UUIDs are generated randomly by our script.
```

Comment:
```suggestion
You'll need to change the actual `uuid` value to a value that exists in your database, because the UUIDs are generated randomly by our script.
```